docs/userguide/icudata.md

```diff
@@ -334,10 +351,10 @@ by their package name. Whenever a data item is requested from a package and that
 package has been loaded before, then the cached package is used immediately
 instead of searching through the filesystem.
 
-*ICU versions before 2.2 always searched data packages before looking for
-individual files, which made it impossible to override packaged data items. See
-the ICU 2.2 download page and the readme for more information about the
-changes.*
+> :point_right: *ICU versions before 2.2 always searched data packages before
```

Comment:
not quite the agreed style -- `> :point_right: **Note**: that ICU for C by default ...`

(and please remove the italics)